helm/ako/crds/ako.vmware.com_aviinfrasettings.yaml

```diff
@@ -29,6 +29,8 @@ spec:
                           type: string
                         cidr:
                           type: string
+                        v6cidr:
```

Comment:
please run `make sync-crd-files` to sync the crd changes with ako-operator

Reply:
done